### PR TITLE
SONAR-7266 Project admin can manage project permissions

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/permission/ws/template/ApplyTemplateAction.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public void define(WebService.NewController context) {
       .setHandler(this);
 
     createTemplateParameters(action);
-    createProjectParameter(action);
+    createProjectParameters(action);
```
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/permission/ws/template/ApplyTemplateAction.java]: It would have been nice to have this renaming done in another commit, as it's a "side" modification.
Moving it in another commit makes commits clearer

File: server/sonar-server/src/test/java/org/sonar/server/permission/ws/template/SearchTemplatesActionTest.java [hunk 2]
```diff
@@ -175,14 +172,6 @@ public void fail_if_not_logged_in() {
   }
 
   @Test
-  public void fail_if_not_global_admin() {
```
Comment (julienlancelot) [inline server/sonar-server/src/test/java/org/sonar/server/permission/ws/template/SearchTemplatesActionTest.java]: Why have you removed this test ? It should just be replaced by a test that check that the user needs to be logged.

**Comment by julienlancelot** [comment]
Description of the WS api/permissions/search_templates must be updated to explain that the user needs to be logged to access the WS

**Comment by julienlancelot** [comment]
Required : Please update the WS description and add the missing UT.
Nice to have : split your modification into 2 commits.
Then LGTM !

File: server/sonar-server/src/test/java/org/sonar/server/permission/ws/template/SearchTemplatesActionTest.java (hunk 2 above)
Comment (teryk) [inline server/sonar-server/src/test/java/org/sonar/server/permission/ws/template/SearchTemplatesActionTest.java]: because there is another test that tests just that

Comment (julienlancelot) [inline server/sonar-server/src/test/java/org/sonar/server/permission/ws/template/SearchTemplatesActionTest.java]: Oups, indeed sorry
